resources/configure_dependencies.py

```diff
@@ -465,14 +465,14 @@ def InstallPETSC(pkg: str, ver: str, gold_file: str):
             log_file.write(f"{command}\n{err}\n")
             package_log_file.write(f"{command}\n{err}\n")
 
-        command = f"make all -j{argv.jobs}"
+        command = f"make OMAKE_PRINTDIR=gmake all -j{argv.jobs}"
```

Comment:
```suggestion
         make_command = "make"
         s, e, outstr = ExecSub(make_command + " --version")
         if outstr.find("GNU Make") > 0:
             make_command = "make OMAKE_PRINTDIR=gmake"
             
        command = f"{make_command} all -j{argv.jobs}"
```
Per @Naktakala  suggestion in Slack